base/abstractarray.jl

```diff
@@ -842,6 +842,7 @@ isempty(a::AbstractArray) = (length(a) == 0)
 
 
 ## range conversions ##
+AbstractVector{T}(r::AbstractRange) where {T} = map(T,r)
```

Comment:
Maybe just call `convert(AbstractVector{T}, r)`? That's a more standard constructor definition than calling `map`.

Reply:
Convert itself falls back to this constructor I think.

Reply:
I guess not, since it currently doesn't exist? :-/